### ISSUE TITLE
Adding ipywidgets without an extra div

### ISSUE THREAD

**Issue by egormkn** [widgetti/ipyvuetify#291]
Hi!
When I use ipywidgets inside of ipyvuetify widget, they are wrapped in a `<div stye="height: 100%;"></div>`. This doesn't allow customizing layout with flexbox, because I can't change the class or style of this wrapper. Is it possible to display ipywidgets without an extra div? It seems that when I put ipywidgets widget inside another ipywidgets widget (VBox), there is no extra div.

Example:

![image](https://github.com/widgetti/ipyvuetify/assets/7170781/fb7068e1-8285-49f9-9aec-1cf6d58e43be)

```python
import ipywidgets as w
import ipyvuetify as v

button = w.Button(description="ipywidgets button")

label = v.Html(tag="span", children=["Example label"])

v_button = v.Btn(children=["Vue Button"])

input = w.BoundedFloatText(
    value=10,
    layout=w.Layout(width="auto", min_width="var(--jp-widgets-inline-width)", flex="1 0 100%"),
)

example = v.Html(
    tag="div",
    style_="display: flex; flex-flow: row nowrap; align-items: center;",
    children=[label, v_button, input],
)

# solara run example.py:example
```

**Comment by egormkn** [comment]
I found the code that creates that div in ipyvue repository. Maybe this issue should be moved there.

https://github.com/widgetti/ipyvue/blob/d307dbdceaa05f01fe22be57d157d02c9ee8fbdf/js/src/VueRenderer.js#L48-L50

**Comment by egormkn** [comment]
I tried to create a comment node in render function, mount a JupyterPhosphorWidget as a sibling to this node and then remove the comment node: https://github.com/widgetti/ipyvue/compare/master...egormkn:ipyvue:master 
It now renders correctly, however I'm not sure if such kind of manual DOM manipulation is safe in terms of Vue reactivity.

Also, in my tests in Jupyterlab I noticed a bug that is caused by that `height: 100%`: the wrapper div takes 100% of the output height.
![image](https://github.com/widgetti/ipyvuetify/assets/7170781/f1d31f03-fef7-41b6-98e5-23d2f62b3949)

**Comment by mariobuikhuizen** [comment]
yeah, the extra div is indeed a pain point, but we have not found a way around it yet.

I've tried your branch and it works for getting an embedded widget in. After I removed `refNode.remove();` I could also replace an embedded widget. But changing the order of widgets of the same parent doesn't work.

Notebook example:
```python
import ipywidgets
import ipyvue as vue

children = [ipywidgets.Button(description="hi1"), ipywidgets.Button(description="hi2")]

main = vue.Html(tag="div", children=children)
main
```

Then change the order:
```python
main.children = children[::-1]
```
Notice the order of the children doesn't change in the DOM.

A workaround for the extra div is to have a div with the right properties in ipyvue and add the embeded widget to that.
